apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisHeader.vue

```diff
@@ -102,36 +75,6 @@ const handleDeleteQueryConfirm = async () => {
     <div class="cost-analysis-header">
         <section class="title-section">
             <p-heading :title="costAnalysisPageStore.selectedQueryId ? state.title : state.defaultTitle">
-                <template #title-left-extra>
```

Comment:
Delete this

<img width="649" alt="스크린샷 2023-09-01 오후 5 04 08" src="https://github.com/cloudforet-io/console/assets/18563857/a2f75a89-8aff-4be1-a687-9642b480d24a">